packages/react-magma-dom/src/inverse/index.ts

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+interface InverseInterface {
+  isInverse?: boolean;
+}
+
+export const InverseContext = React.createContext<InverseInterface>({
+  isInverse: false,
+});
+
+export function getIsInverse(
```

Comment:
we can probably use the context here instead of making the devs pass it in

Reply:
Great feedback!  Will change this.